catalyst-gateway/bin/src/event_db/legacy/queries/event/config.rs

```diff
@@ -0,0 +1,81 @@
+//! Config Queries
```

Comment:
Why these queries is a part of the `legacy` code ?

Reply:
I moved queries out of legacy. But have kept the trait for now as I have another branch which I want to merge after this and it will conflict with everything. I'll refactor the trait stuff in two PRs.